src/lib.rs

```diff
@@ -147,6 +147,62 @@
 //! See the [`platform`] module for documentation on platform-specific cargo
 //! features.
 //!
+//! # Platform/Architecture Support
+//!
+//! Platform support on `winit` has two tiers: Tier 1 and Tier 2.
+//!
+//! - Tier 1 is **guaranteed to work**. Targets in this tier are actively tested both in CI and by
+//!   maintainers.
+//! - Tier 2 is **guaranteed to build**. Code compilation is tested in CI, but deeper testing is not
+//!   done.
```

Comment:
I'm a bit confused, we don't actually test compilation of all these tier 2 targets in CI.
In the OP you said "we hope" compiles, which might be more appropriate here, but is probably very weird to add here.

Reply:
The goal is to start doing that in CI after we finalize this list.

Reply:
Sure!